ogr/services/github/service.py

```diff
@@ -76,6 +85,16 @@ def github_app_private_key(self):
 
         return None
 
+    @property
+    def token_manager(self) -> GithubTokenManager:
+        if self._token_manager is None:
+            self._token_manager = (
+                TokmanGithubTokenManager(self.tokman_instance_url)
+                if self.tokman_instance_url
+                else OgrGithubTokenManager(self)
+            )
+        return self._token_manager
```

Comment:
What about basic token-based authentication. Don't we want to ignore those managers (return `None`?) in that case?